wire-ios/Wire-iOS/Sources/UserInterface/CustomAppLock/Setup/PasscodeSetupViewController.swift

```diff
@@ -96,15 +94,21 @@ final class PasscodeSetupViewController: UIViewController {
 
     private let useCompactLayout: Bool
 
-    private lazy var infoLabel: UILabel = {
-        let style = DownStyle.infoLabelStyle(compact: useCompactLayout)
-        let label = UILabel()
-        label.configMultipleLineLabel()
-        label.attributedText = .markdown(from: context.infoLabelString, style: style)
+    private let infoLabel: UILabel = {
```

Comment:
Optional: I think it is good to keep this & `scrollView`  as `lazy` for UIViewControllers. This is because `UIViewControllers` are designed to be lazy to save memory - E.g. when we create a view controller its subviews are not created. It is only when we actually display the view controller that its subviews are created.

However, I see that we are calling `setupViews` in the `init` method which is already breaking view controller laziness. So if you choose to change this then `setupViews` should be moved to `viewDidLoad`.

Reply:
done